themes/mlibrary/custom.php

```diff
@@ -13,6 +13,23 @@
  * Extends featured exhibit function to include snippet from description for exhibits and
  * read more link
  **/
+
+/**
+* Create the information in cards in introduction page, image with title and text
+*
+**/
+function mlibrary_display_exhibit_card_info($rawAttachment,$block,$exhibitPage)
+{
+   $file = $rawAttachment[0]->getFile();
+   $page_image = file_image('thumbnail','', $file);
+   $page_title = metadata($exhibitPage, 'title').'<br>';
```

Comment:
`<br>` is usually bad for maintainability, is there a way to get the layout you want using css?  Either add a bottom margin to a thing, or make a thing that's `display:inline-block;` into `display: block;`.